### DISCUSSION TITLE
Database - fancy documentation for WIKI

### DISCUSSION THREAD

**Discussion by frailefeijoo** [dbeaver/dbeaver#19483] [Q&A]
Hi, i need to document a complex database in platform that suppor markdown language like github.

i will like know the best way to make this dynamically only using GIT and SQL editor. 
Currently i work with param LAST_WIKI_LAUCNH to determine how the DBA_OBJECTS can be listed since last DML or DDL mutation, so i recived the database information in custom select statment and export it to markdown. 

In practic the problem is how export big bulk of dynamic data using client-side only. I try using the `@export`  command but need to thers helpfull for implements the outputFolder dynamically (outputFolder attribute). 

- [related problem](https://github.com/dbeaver/dbeaver/issues/14680#issuecomment-1484150106)

am I must to be try other way? is command syntax provie quickly solution?

Thanks!

**Comment by frailefeijoo** [comment]
![image](https://user-images.githubusercontent.com/45419986/228564008-45fd4d8d-846c-4857-b47e-cf3ac61345a2.png)

Hello again! Custom or declared variable, ${DECLARED_VAR} and others not works in comunity edition. Is only reserved for Enterprise?

**Comment by frailefeijoo** [comment]
The attributes `useSingleFile` and `appendToFile` not working if you dont use the wizard

**Comment by E1izabeth** [comment]
Maybe you found some bug(s). Could you create an issue?

**Reply by frailefeijoo** [reply]
This discussion must continue open for searching any util response 
However i created [issue](https://github.com/dbeaver/dbeaver/issues/19690) for describe a context and brind more information 

Thanks for the advice!
